en/config/actuators.md

```diff
@@ -1,96 +1,242 @@
+
+An "example" control surface section for a vehicle with two ailerons is shown below.
+Note that ailerons only affect roll, so the pitch and yaw fields are disabled.
+
+![Control Surface Setup Example](../../assets/config/actuators/control_surfaces_geometry.png)
+
+The fields are:
+
+- `Control Surfaces`: The number of control surfaces (set this first!)
+- `Type`: The type of each control surface: `LeftAileron`, `RightAileron`, `Elevator`, `Rudder`, `Left Elevon`, `Right Elevon`, `Left V-Tail`, `Right V-Tail`, `Left Flap`, `Right Flap`, `Airbrakes`, `Custom`.
+- `Roll scale`: Roll scale (normalised: -1 to 1)
+- `Pitch scale`: Pitch scale (normalised: -1 to 1)
+- `Yaw scale`: Yaw scale (normalised: -1 to 1)
+- `Trim`: ?
+- `Skew`: ?
+- `Lock control surfaces in hover`: `Enabled` or `Disabled`.
+
+The scaling values are normalised (-1 to 1), decoupling the outputs of PX4 controllers from the physical servo control signals.
+
+
+#### Control Surface Deflection Convention
+
+The diagram below shows the convention for deflections:
+
+![Control Surface Deflections](../../assets/config/actuators/plane_control_surface_convention.png)
+
+In summary:
+- **Horizontal Control Surfaces:** Upwards movement equals positive deflection.
+  Includes Ailerons, ...
+- **Vertical Control Surfaces:** Rightwards movement is positive deflection
+  Includes rudders etc.
+- **Mixed Control Surfaces:** Upwards/rightwards movement is positive (as above)
+  Includes V-Tail etc.
+
+
+### Motor Tilt Servo Geometry
+
+[VTOL tiltrotor vehicles](../frames_vtol/tiltrotor.md) can tilt their motors to transition between hover and forward flight.
+This section defines the properties of the tilting servos.
+These are mapped to specific motors in the [motor geometry for a tiltrotor](../config/actuators.md#motor-geometry-vtol-tiltrotor).
+
+The example below shows the tilt servo setup for the [tiltrotor motor geometry shown above](../config/actuators.md#motor-geometry-vtol-tiltrotor).
+
+![Tilt Servo Geometry Setup Example](../../assets/config/actuators/tilt_servo_geometry_config.png)
+
+The values that can be set are:
+- `Tilt servos`: The number of servos (tiltable motors).
+- `Angle at min tilt`: [Maximum tilt angle](#tilt-servo-coordinate-system), relative to the z axis (degrees).
+- `Angle at max tilt`: [Minimum tilt angle](#tilt-servo-coordinate-system), relative to the z-axis (degrees).
+- `Tilt direction`: `Towards front` (Positive X direction) or `Towards right` (Positive Y direction).
+- `Use for control`: [Tilt server used for yaw/pitch](#tilt-servos-for-yaw-pitch-control)
+  - `None`: Torque control is not used.
+  - `Yaw`: Tilt servos used to control yaw.
+  - `Pitch`: Tilt servos used to control pitch.
+  - `Both Yaw and Pitch`: Tilt servos are used to control both yaw and pitch.
+
+
+#### Tilt Servo Coordinate System
```

Comment:
@bkueng This is how I think tilt servo values work. This is linked to  from section above. The approach is "by example" - which I think removes the ambiguity.